tests/integration/targets/azure_rm_adapplication/tasks/main.yml

```
@@ -103,5 +120,4 @@
 
 - name: Assert there is no application
   ansible.builtin.assert:
-    that:
-      - output.applications | length == 0
+    that: output.applications | length == 0
```

The error( No new line character at the end of file)  still exsit, Please '
```suggestion
    that:
      output.applications | length == 0
```